src/mixins/_spacing.scss

```diff
@@ -36,12 +36,12 @@ $fluid-gap: fluid.fluid-clamp(
   $fluid-spacing-breakpoint-max
 );
 
-@mixin fluid-padding-block() {
-  padding-block: $fluid-spacing-block;
+@mixin fluid-padding-block($factor: 1) {
+  padding-block: calc($fluid-spacing-block * $factor);
```

Comment:
Instead of wrapping this in a second `calc` function, could we do something like...

```scss
padding-block: fluid.fluid-clamp(
  $fluid-spacing-block-min * $factor,
  $fluid-spacing-block-max * $factor,
  $fluid-spacing-breakpoint-min,
  $fluid-spacing-breakpoint-max
);
```

(Same for the inline version but with the inline values instead.)

Reply:
Excellent call, @tylersticka, done! https://github.com/cloudfour/cloudfour.com-patterns/pull/2044/commits/b5acd1673f4bf400174803eb1a9d2d6f5ed97cc3